Source/WebKit/UIProcess/API/Cocoa/_WKActivatedElementInfo.h

```diff
@@ -48,6 +48,7 @@ WK_CLASS_AVAILABLE(macos(10.10), ios(8.0))
 @property (nonatomic, readonly) CGRect boundingRect;
 @property (nonatomic, readonly) NSString *ID WK_API_AVAILABLE(macos(10.12), ios(10.0));
 @property (nonatomic, readonly) BOOL isAnimatedImage WK_API_AVAILABLE(macos(10.15), ios(13.0));
+@property (nonatomic, readonly) BOOL isUsingAlternateURLForImage;
```

Comment:
This should probably be in the Internal header.

Reply:
@pxlcoder how do I do this? When I try to do it it just complains about `__isUsingAlternateURLForImage` not being a thing

Reply:
(looks you figured this out – `_isUsingAlternateURLForImage` and implementing the getter is the right fix)